File: app/src/main/java/com/jeeldobariya/passcodes/utils/ExampleTestableCode.java
package com.jeeldobariya.passcodes.utils;

class ExampleTestableCode {
    int checkStrength(String password) {
        if (password == null || password.isEmpty()) {
            return -1;
        }

        int length = password.length();

        if (length < 8) {
            return 0;
        } else {
            return 1;
        }
    }
}

File: app/src/main/kotlin/com/jeeldobariya/passcodes/utils/ExampleTestableCode.kt
package com.jeeldobariya.passcodes.utils

class ExampleTestableCode {
    fun checkStrength(password: String): Int {
        if (password.isEmpty()) {
            return -1;
        }

        val length = password.length;

        if (length < 8) {
            return 0;
        } else {
            return 1;
        }
    }
}
